### ISSUE TITLE
CodeMirror 0.50.2 bug

### ISSUE THREAD

**Issue by karlwessel** [fonsp/Pluto.jl#72]
This is a strange one...

When I have the following notebook (which is not the real code but a meaningless MWE):
```
### A Pluto.jl notebook ###
# v0.5.20
# ╔═╡ ff8b2572-7bbb-11ea-3513-175ab56e1fd7
x = [1,2]

# ╔═╡ 0ffef548-7bba-11ea-3b0c-511048e9d2f4
[(x[2])]

# ╔═╡ Cell order:
# ○ ff8b2572-7bbb-11ea-3513-175ab56e1fd7
# ○ 0ffef548-7bba-11ea-3b0c-511048e9d2f4
```
and in the second cell try to delete the `[` between the `x` and the `2` in `[(x[2])]` the browser tab hangs. Always.

The first cell actually doesn't matter. The same also happens if the notebook only contains the second cell. I only added it to demonstrate that the code is ok.

The bracket combination in the second cell seems to actually matter somewhat. At least I wasn't able to remove any of them to reduce the MWE further.

Although this might seem like a corner case, it actually happened more than once now for me while working with actual notebooks.

This happens with Chromium and Firefox.

**Comment by fonsp** [comment]
I'll look into this right now!

**Comment by fonsp** [comment]
Unfortunately the problem is in the [CodeMirror formatter](https://github.com/codemirror/CodeMirror/blob/master/mode/julia/julia.js) (it also breaks in a [standalone editor](https://codemirror.net/mode/julia/index.html)), I'll try to fix it and send them a PR.

**Comment by fonsp** [comment]
I've [submitted an Issue with CodeMirror](https://github.com/codemirror/CodeMirror/issues/6213) and downgraded CodeMirror to `5.50.0` as a temporary fix (in the upcoming Pluto `0.5.21`).

**Comment by fonsp** [comment]
Marijn fixed the CodeMirror bug! Will be fixed here when CodeMirror releases their next version.
